WTWSMS/common/cb_types/WtWSMS_migration_unification_cb_types.txt

```diff
@@ -39,12 +39,23 @@ migration = {
 				ROOT = {
 					OR = {
 						religion = tengri_pagan
+						religion = tengri_pagan_reformed
 						religion = slavic_pagan
+						religion = slavic_pagan_reformed
 						religion = germanic_pagan
+						religion = germanic_pagan_reformed
 						culture = alan
+						is_nomadic = yes
 					}
 					NOT = { has_character_modifier = launched_migration_war }
-					NOT = { higher_tier_than = KING } # No Emperors
+
+					NOT = {
+						AND = {
+							higher_title_than = KING
+							is_nomadic = no
+						}
+					}
+	
 					NOT = { culture = FROM }
 					independent = yes # No vassals
 					war = no # Ensures that the events activate correctly  
```

Comment:
I think the current setup should be enough for 1 to not be an issue. On 2, that is a design question. I disagree with you and think reformed pagans are more settled than the unreformed versions.